squadutilslib.py

```diff
@@ -229,3 +231,63 @@ def create_custom_reproducer(
 def create_ltp_custom_command(tests):
     return f"cd /opt/ltp && ./runltp -s {' '.join(tests)}"
 
+
+def tuxtest_to_tuxplan_entry(tuxsuite_test):
+    tuxsuite_test = re.sub("tuxsuite test submit", "", tuxsuite_test)
+    split_params = tuxsuite_test.split(" --")
+
+    dict_entry = dict()
+    for item in split_params:
+        params_list = re.findall(r"""(\S+) (.+)""", item)
+        for params in params_list:
+            key, value = params
+            if key == "timeouts":
+                test, timeout = value.split("=")
+                if key in dict_entry:
+                    dict_entry[key][test] = int(timeout)
+                else:
+                    dict_entry[key] = dict()
+                    dict_entry[key][test] = int(timeout)
+            elif key in dict_entry:
+                if not isinstance(dict_entry[key], list):
+                    dict_entry[key] = [dict_entry[key]]
+                dict_entry[key].append(value)
+            elif key == "commands":
+                value = re.sub(r"""['"]+""", r"", value)
+                dict_entry[key] = [value]
+                dict_entry["command-name"] = value
+            elif key == "overlay":
+                dict_entry[key] = [value]
+            else:
+                dict_entry[key] = value
+
+    return dict_entry
+
+
+def create_tuxsuite_plan_from_tuxsuite_tests(tuxtest_filename, plan_name):
+    tuxrun_list = open(tuxtest_filename).read().splitlines()
```

Comment:
tuxrun_list is missleading since you grab a tuxtest reproducer right?

Reply:
Renamed to tuxtest_list